aviary/interface/build_model.py

```diff
@@ -0,0 +1,692 @@
+import csv
+import math
+from tkinter import *
+import tkinter as tk
+import tkinter.ttk as ttk
+from tkinter import messagebox
+from tkinter import filedialog
+import aviary.api as av
+from copy import deepcopy
+
+
+data = deepcopy(av.CoreMetaData)
+list_values = list(data.values())
+list_keys = list(data.keys())
+list_len = len(list_values)
+root = Tk()
+width_window = 1200
+height_window = 800
+root.geometry("%dx%d" % (width_window, height_window))
+root.title("Model Aircraft Input")
+
+headers = []
+name_each_subhead = []
+entries_per_subhead = []
+old_subhead = ''
+for key in list_keys:
+    if ':' in key:
+        subhead = key.split(':')[1]
+        header = key.split(':')[0]
+        if subhead == old_subhead:
+            entries_per_subhead[-1] += 1
+        else:
+            entries_per_subhead.append(1)
+            name_each_subhead.append(subhead)
+            headers.append(header)
+        old_subhead = subhead
+    else:
+        entries_per_subhead.append(1)
+        name_each_subhead.append(key)
+        headers.append('dynamic')
```

Comment:
Dynamic variables (all those that are `Dynamic.Mission.<variable_name>` are not actually used in the .csv file. Because of this, I don't think they need to be included in this GUI.

There could be value in having the dynamic variables present in the GUI and you then output those values as initial guesses used in the `phase_info` object, but I think that capability is already represented by the `draw_mission` tool. So, I'd suggest removing the dynamic variables from this GUI unless I'm missing a reason (which I easily could be!)